api/services/kakaoService.js

```diff
@@ -0,0 +1,29 @@
+const axios = require('axios')
+
+const getAccessTokenByKakao = async(code, res) => {
```

Comment:
두번째 argument `res`는 불필요하므로 삭제하는 것이 좋겠습니다.
```suggestion
const getAccessTokenByKakao = async(code) => {
```